openedx/core/djangoapps/edx_global_analytics/utils/utils.py

```diff
@@ -18,31 +18,40 @@
 logger.setLevel(logging.INFO)
 
 
-def fetch_instance_information(name_to_cache, query_type, activity_period, cache_timeout=None):
+def fetch_instance_information(query_type, activity_period, name_to_cache=None):
     """
     Calculate instance information corresponding for particular period as like as previous calendar day and
     statistics type as like as students per country after cached if needed.
     """
+    query_result = get_query_result(query_type, activity_period)
```

Comment:
we should pass to cache_instance_data not query result, but query name from dict.